src/styles/index.scss

```diff
@@ -1,3 +1,197 @@
+* {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+/* #region header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 23px 0;
+  display: block;
+}
+
+.is-active {
+  color: $active-color;
```

Comment:
The variable `$active-color` is used here, but it is not defined within this file. Ensure that `$active-color` is defined in a file that is imported before this one, or define it here to avoid potential errors.